### PR TITLE
feat: create velocity command.

### PR DESCRIPTION
**Pull request by rodrigoleonello**

### CODE REVIEW

File: tasks/MavDroneControlTask.cpp [hunk 1]
```diff
@@ -116,6 +136,13 @@ bool MavDroneControlTask::startHook()
     if (!MavDroneControlTaskBase::startHook())
         return false;
 
+    // Send it once before starting offboard, otherwise it will be rejected.
+    const Offboard::VelocityNedYaw stay{};
+    mOffboard->set_velocity_ned(stay);
+    Offboard::Result offboard_result = mOffboard->start();
+    if (offboard_result != Offboard::Result::Success)
+        return false;
```
Comment (jhonasiv) [inline tasks/MavDroneControlTask.cpp:144]: Shouldnt offboard mode only start when we use velocity control and it's not started yet?

Comment (jhonasiv) [reply tasks/MavDroneControlTask.cpp:144]: Nevermind, offboard mode allows for both position and velocity control

Comment (rodrigoleonello) [reply tasks/MavDroneControlTask.cpp:144]: Actually I decided to push the code to discuss some points, and  that is one of them. I'm not sure about that behavior...your ideia sound good (it's like the mission).

Comment (jhonasiv) [reply tasks/MavDroneControlTask.cpp:144]: I guess we could limit it to position and velocity commands, you can check if it was started before and if it wasnt start it. Did you check on simulation if doing it like this actually messes with other commands like takeoff or mission?

Comment (rodrigoleonello) [reply tasks/MavDroneControlTask.cpp:144]: > Did you check on simulation if doing it like this actually messes with other commands like takeoff or mission?
no yet....

**Review by jhonasiv — APPROVED** [review]

File: tasks/MavDroneControlTask.hpp [hunk 2]
```diff
@@ -144,34 +143,40 @@ argument.
...
+            std::unique_ptr<mavsdk::Offboard> const& offboard,
+            drone_dji_sdk::VehicleSetpoint const& setpoint);
 
-        void landingCommand(std::unique_ptr<mavsdk::Telemetry> const& telemetry,
-                            std::unique_ptr<mavsdk::Action> const& action,
-                            std::unique_ptr<mavsdk::Offboard> const& offboard,
-                            drone_dji_sdk::VehicleSetpoint const& setpoint);
+        bool velCommand(
+            std::unique_ptr<mavsdk::Offboard> const& offboard,
+            drone_dji_sdk::VehicleSetpoint const& setpoint);
```
Comment (jhonasiv) [inline tasks/MavDroneControlTask.hpp:175]: There isnt a drone_dji_sdk::VehicleSetpoint anymore

**Pull request merged**